doc.config.json

```diff
@@ -63,6 +63,16 @@
               }
             ]
           },
+          {
+            "title": "Guides",
+            "disableAutoPrefix": true,
+            "routes": [
+              {
+                "title": "Build a simple user profile API",
+                "path": "/guides/profiles.md"
```

Comment:
This is not valid, wait until supabase PR is merged before merging this one.

Reply:
merged now